net/bluetooth/bluetooth_sockif.c

```diff
@@ -904,9 +906,9 @@ static ssize_t bluetooth_sendto(FAR struct socket *psock,
 {
   ssize_t ret;
 
-  /* Only SOCK_RAW on L2CAP is supported */
+  /* Only SOCK_RAW is supported */
 
-  if (psock->s_type == SOCK_RAW && psock->s_proto == BTPROTO_L2CAP)
+  if (psock->s_type == SOCK_RAW)
```

Comment:
We should break this out into the different sockets here like I did for other ones instead of further changing `psock_bluetooth_sendto`  i.e.
`psock_bluetooth_sendto` -> `psock_bluetooth_l2cap_sendto`  `psock_bluetooth_hci_sendto`

There are a lot of difference in these functions especially when you start adding the channels.

Reply:
I chose to use just `psock_bluetooth_sendto` for both since right now there's a lot of common code, but if it gets bigger with the other channels we can split it.